### DISCUSSION TITLE
dlg.py  DLG test is not mentioned the Pylinac documentaion

### DISCUSSION THREAD

**Discussion by DShankar369** [jrkerns/pylinac#509] [Q&A]
Could you please provide documentation or a demo file plot for dlg.py in the Pylinac 3.25 documentation? It would be extremely helpful.

**Comment by jrkerns** [comment]
That's correct. It's purposely undocumented for the time being because it's not really for rollout.

**Reply by crcrewso** [reply]
Do you have sample plans for DLG?

**Reply by jrkerns** [reply]
I'm waiting on our product manager to take some sample images with a few different plan variations to see what sticks/is most robust. The goal of the DLG module will be to do both single-image DLG for easy constancy at higher frequency and a 2D DLG measurement for a by-leaf and off-axis 2D distribution. Although what with Eclipse now finally modelling leaf ends I'm not sure if the utility will be as appealing; we'll see. So, no, no sample plans at the moment, but in the next quarter we should have something and it will get rolled into the plan generator module.
